ocaml/idl/datamodel_host.ml

```diff
@@ -1459,6 +1459,18 @@ let host_query_ha = call ~flags:[`Session]
     ~allowed_roles:_R_POOL_ADMIN
     ()
 
+  let copy_primary_host_certs = call
+    ~name:"copy_primary_host_certs"
+    ~in_oss_since:None
+    ~in_product_since:rel_next
+    ~doc:"useful for secondary hosts that are missing some certs"
+    ~params:[
+      Ref _host, "host", "this host receives a copy of the primary host's trusted certificates";
+    ]
+    ~allowed_roles:_R_POOL_ADMIN
+    ~hide_from_docs:true
```

Comment:
If this call is only to be used within the pool, then you can add `~pool_internal:true`. This show up in the `~intra_pool_only` argument to `Session_check.check`, which is used by the API server to authenticate incoming calls.